### ISSUE TITLE
Calling `setVariables` on an unsubscribed `ObservableQuery` should not trigger a fetch

### ISSUE THREAD

**Issue by tmeasday** [apollostack/apollo-client#1077]
See https://github.com/apollostack/react-apollo/issues/381.

For example, if you run

```js
query = client.watchQuery(...);
// note no subscribe, so no fetch has happened

// it wouldn't make sense to fetch here
query.setVariables(..);
```

I'll write up a test and fix this when I have a chance.

**Comment by tmeasday** [comment]
test

**Comment by tmeasday** [comment]
Fixed by #1154
